packages/test/test-version-utils/src/describeCompat.ts

```diff
@@ -38,14 +39,19 @@ function createCompatSuite(
 		apis: CompatApis,
 	) => void,
 	compatFilter?: CompatKind[],
+	minVersion?: string,
 ): (this: Mocha.Suite) => void {
 	return function (this: Mocha.Suite) {
 		let configs = configList.value;
 		if (compatFilter !== undefined) {
 			configs = configs.filter((value) => compatFilter.includes(value.kind));
 		}
-
+		const versionsMap = getAllFluidVersions();
 		for (const config of configs) {
+			if (minVersion && isCompatVersionBelowMinVersion(minVersion, versionsMap, config)) {
+				// skip current config if compat version is below min version supported for test suite
+				continue;
```

Comment:
There is an optimization we could make here given that we are filtering tons of configs, here and lines above with compatFilter. mochaGlobalSetup ensures all versions got from the configList are installed when there is no need for some of the filtered configs. Changes could be considerable so maybe it could be made in a follow up PR/ task